instance/tests/api/test_instance.py

```diff
@@ -28,9 +28,12 @@
 
 from instance import github
 from instance.models.instance import SingleVMOpenEdXInstance
-from instance.models.server import OpenStackServer
 from instance.tests.api.base import APITestCase
-from instance.tests.models.factories.instance import SingleVMOpenEdXInstanceFactory
+from instance.tests.models.factories.instance import (
+    ConfiguringSingleVMOpenEdXInstanceFactory, ConfigFailedSingleVMOpenEdXInstanceFactory,
+    ErrorSingleVMOpenEdXInstanceFactory, RunningSingleVMOpenEdXInstanceFactory,
+    SingleVMOpenEdXInstanceFactory, TerminatedSingleVMOpenEdXInstanceFactory, WaitingSingleVMOpenEdXInstanceFactory
```

Comment:
Optional: Also, it would arguably be nicer to just import `SingleVMOpenEdXInstanceFactory` and `Status` and then use things like `SingleVMOpenEdXInstanceFactory(status=Status.Configuring)` instead of defining a separate class for each status.